freqtrade/freqtradebot.py

```diff
@@ -937,7 +937,9 @@ def get_valid_enter_price_and_stake(
             # Don't call custom_entry_price in order-adjust scenario
             custom_entry_price = strategy_safe_wrapper(self.strategy.custom_entry_price,
                                                        default_retval=enter_limit_requested)(
-                pair=pair, current_time=datetime.now(timezone.utc),
+                pair=pair,
+                trade=trade,  # type: ignore[arg-type]
```

Comment:
There's no reason to ignore this here if typehints are correct (using `trade: Trade` is incorrect - as that implies the value cannot be None, which is incorrect).